lib/grammars.coffee

```diff
@@ -200,10 +203,10 @@ module.exports =
         args: (context) -> ['-c', "xcrun clang++ -fcolor-diagnostics -Wc++11-extensions -Wall -include stdio.h -include iostream -framework Cocoa " + context.filepath + " -o /tmp/objc-cpp.out && /tmp/objc-cpp.out"]
 
   ocaml:
-    "File Based":
-      command: "ocaml"
-      args: (context) -> [context.filepath]
-
+      "File Based":
+        command: "ocaml"
```

Comment:
Looks like some extra whitespace changes. I'll go ahead and test with these and may remove them in a later commit if you don't get to it first.